tasks/main.yml

```diff
@@ -34,6 +34,7 @@
   when: >
     (ansible_distribution == 'Ubuntu' and ansible_distribution_version.split(".")[0]|int >= 16) or
     (ansible_distribution == 'Debian' and ansible_distribution_version.split(".")[0]|int >= 8) or
+    (ansible_distribution == 'RedHat' and ansible_distribution_version.split(".")[0]|int >= 7) or
```

Comment:
Tested on both REHL7 and CentOS 7. It works  like a charm.
Thanks for the quick hint :+1: